.github/styles/cloudflare/BrandNames.yml

```
@@ -0,0 +1,16 @@
+---
+# Warning: cloudflare.BrandNames
+#
+# Suggests correct spelling and capitalization for brand names.
+#
+# For a list of all options, see https://vale.sh/docs/topics/styles/
+
+extends: substitution
+message: "**Warning**: Use correct brand name: '%s' instead of '%s'."
+level: warning
+ignorecase: false
+swap:
+  (?!Cloudflare)([cC]loud[fF]lare): Cloudflare
+  (?!cdnjs)([cC][dD][nN]\.?[jJ][sS]): cdnjs
+  (?!GitHub)([gG]it\s?[hH]ub): GitHub
+  (?!GitLab)([gG]it\s?[lL]ab): GitLab
```

```suggestion
  (?!Cloudflare)(?:[cC]loud[fF]lare): Cloudflare
  (?!cdnjs)(?:[cC][dD][nN]\.?[jJ][sS]): cdnjs
  (?!GitHub)(?:[gG]it\s?[hH]ub): GitHub
  (?!GitLab)(?:[gG]it\s?[lL]ab): GitLab
```
Per the CI run, capture groups aren't supported: https://github.com/cloudflare/cloudflare-docs/actions/runs/9689862423/job/26738869158#step:5:50